files/default/sql/ddl/1.1.0__initial_tables.sql

```diff
@@ -1112,6 +1112,51 @@ CREATE TABLE `project_team` (
 ) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- table structure for table `schemas`
+--
+
+/*!40101 set @saved_cs_client     = @@character_set_client */;
+/*!40101 set character_set_client = utf8 */;
+CREATE TABLE `schemas` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `schema` varchar(10000) COLLATE latin1_general_cs NOT NULL,
+  `project_id` int(11) NOT NULL,
+  PRIMARY KEY (`id`, `project_id`),
+  CONSTRAINT `project_idx` FOREIGN KEY (`project_id`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+--
+-- table structure for table `subjects`
+--
+
+/*!40101 set @saved_cs_client     = @@character_set_client */;
+/*!40101 set character_set_client = utf8 */;
+CREATE TABLE `subjects` (
```

Comment:
How is sharing of schema (or subjects?) implemented in Hopsworks? Maybe I'm too tired, but I don't see how the is this schema (or subject?) shared with project x - is stored in the database.

Reply:
good call. it's not implemented atm because up to now the schemas were global. one can share a kafka topic with another project and everyone can use it but the second user would not have access to the topic's schema.
@tkakantousis, can we put it in another jira or should it be implemented in this one?